01.multithreading/MultiThreading.Task4.Threads.Join/BusinessLayer.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Threading;
+
+namespace MultiThreading.Task4.Threads.Join
+{
+    internal static class BusinessLayer
+    {
+        private static readonly SemaphoreSlim Semaphore = new (1);
```

Comment:
Naming convention is violated in this line

Reply:
done